installer_scripts/install-voyager-airgapped.sh

```diff
@@ -150,8 +150,12 @@ install_perl_module() {
     local requirement_type="$2"
     local required_version="$3"
     local package="$4"
-    
-    echo "Installing module $module_name..."
+
+    # Check if the module is already installed and meets the version requirements
+    check_perl_module_version "$module_name" "$requirement_type" "$required_version" "true"
+    if [[ $? -eq 0 ]]; then
+        return
+    fi
     
     # Extract the package
     tar -xzvf "$package" 1>&2 || { echo "Error: Failed to extract $package"; exit 1; }
```

Comment:
What happens if a module is already installed with a non-required version? How does our installation work? Will it update it to the required version?

Reply:
Hey so it will overwrite any packages that are installed at the expected locations i.e. the paths related to perl etc. However, there might be an edge case where the user has installed modules at an unusual location. Then these commands won't replace the already installed packages. This is an expected edge case in any form of installation I believe.